source/includes/code-examples/find-many/FindManyAsync.cs

```diff
@@ -0,0 +1,77 @@
+using MongoDB.Bson;
+using MongoDB.Bson.Serialization.Attributes;
+using MongoDB.Bson.Serialization.Conventions;
+using MongoDB.Driver;
+using MongoDB.Driver.Linq;
+using static System.Console;
+
+namespace CSharpExamples.UsageExamples.FindMany;
+
+public class FindManyAsync
+{
+    private static IMongoCollection<Restaurant> _restaurantsCollection;
+    private static string _mongoConnectionString = "<Your MongoDB URI>";
```

Comment:
this is not changed

Reply:
D'oh! VS Code search functionality isn't serving me well today.